prep-javascript-variables/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>JavaScript Variables</title>
+  </head>
+  <body>
+    <script>
+      const fullName = 'Shea Pone';
+      console.log('Shea Pone');
```

Comment:
Instead of logging `'Shea Pone'`, you should be logging your `fullName` variable. Since you already assigned this string to the `fullName` variable, logging the variable gets you the same result!